contributing.md

```diff
@@ -25,16 +25,12 @@ Set Android environment variables:
 Create a feature branch to make your changes:
 
     git checkout -b my-feature-name
-    
-Copy all required Android dependencies to your local Maven repository:
-
-    ./scripts/install-dependencies.rb
 
-Perform a full build of all shadows:
+Robolectric is built using Gradle. Both IntelliJ and Android Studio can import the top-level build.gradle file and will automatically generate their project files from it.
 
-    ./gradlew clean assemble install compileTest
+Robolectric supports running tests against multiple Android API levels. The work it must do to support each API level is slightly different, so its shadows are built separately for each. To build shadows for every API version, run:
```

Comment:
Can you remove the sentence `The work it must do to support each API level is slightly different, so its shadows are built separately for each.`. I think that applies to a much older version of Robolectric that actually used java templates to have separate shadow files per api level.